contrib/99designs/gqlgen/tracer.go

```diff
@@ -104,12 +104,12 @@ func (t *gqlTracer) Validate(_ graphql.ExecutableSchema) error {
 func (t *gqlTracer) InterceptOperation(ctx context.Context, next graphql.OperationHandler) graphql.ResponseHandler {
 	opCtx := graphql.GetOperationContext(ctx)
 	span, ctx := t.createRootSpan(ctx, opCtx)
-	ctx, req := graphqlsec.StartRequestOperation(ctx, nil /* root */, span, types.RequestOperationArgs{
```

Comment:
I removed a parameter to the function `StartRequestOperation` because we always passed `nil` as a second argument to it. [Romain also noted this here](https://github.com/DataDog/dd-trace-go/pull/2761#discussion_r1662028234)